app/views/activity_logs/_activity_logs_table.haml

```diff
@@ -1,19 +1,20 @@
-%table.table.table-responsive.w-100
-  %thead
-    %tr
-      %th= t('activity_logs.path')
-      %th= t('activity_logs.http_method')
-      %th= t('activity_logs.email')
-      %th= t('activity_logs.role')
-      %th= t('activity_logs.program')
-      %th= t('activity_logs.http_format')
-      %th= t('activity_logs.http_status')
-      %th= t('activity_logs.payload')
-      %th= t('activity_logs.remote_ip')
-      %th= t('shared.created_at')
-  %tbody
-    = render @activity_logs
+.table-responsive
+  %table.table.w-100
+    %thead
+      %tr
+        %th= t('activity_logs.path')
+        %th= t('activity_logs.http_method')
+        %th= t('activity_logs.email')
+        %th= t('activity_logs.role')
+        %th= t('activity_logs.program')
+        %th= t('activity_logs.http_format')
+        %th= t('activity_logs.http_status')
+        %th= t('activity_logs.payload')
+        %th= t('activity_logs.remote_ip')
+        %th= t('shared.created_at')
+    %tbody
+      = render @activity_logs
 
 - if @pagy.pages > 1
-  .pagy-wrapper
-    != pagy_bootstrap_nav @pagy
\ No newline at end of file
+  .pagy-wrapper.justify-content-lg-end.justify-content-center
+    = pagy_bootstrap_compact_nav @pagy
```

Comment:
@radinreth please check why here is using = pagy_bootstrap_compact_nv @page and the rest of other view still keep using != pagy_bootstrap_nav @pagy